activerecord/lib/active_record/validations.rb

```diff
@@ -62,6 +62,8 @@ def save!(**options)
     #
     # If the argument is +false+ (default is +nil+), the context is set to <tt>:create</tt> if
     # {new_record?}[rdoc-ref:Persistence#new_record?] is +true+, and to <tt>:update</tt> if it is not.
+    # If the argument is an array of contexts, +post.valid?([:create, :update])+, the validations are
+    # run within multiple contexts.
```

Comment:
```suggestion
    # If the argument is an array of contexts, <tt>post.valid?([:create, :update])</tt>, the validations are
    # run within multiple contexts.
```

The `+code+` syntax in RDoc really only works well with simple keywords, FYI.

Reply:
Thanks @zzak I've read up on that and it all makes sense.

I've updated the PR as you requested.